GEN-845

Thabo-hub opened the pull request:
'IM - Logbook - Approve Logbook -  button (Approve Logbook / Unapprove Logbook)   doesn’t change the colour, fixed now color is changing according to the status if approved color is green if not approved color is red'

Thabo-hub committed:
GEN-845

'IM - Logbook - Approve Logbook -  button (Approve Logbook / Unapprove Logbook)   doesn’t change the colour, fixed now color is changing according to the status if approved color is green if not approved color is red'

digesh-bajracharya reviewed (commented):
Same as in another PR you created.

Thabo-hub committed:
GEN 845

'IM - Logbook - Approve Logbook -  button (Approve Logbook / Unapprove Logbook)   doesn’t change the colour, fixed now color is changing according to the status if approved color is green if not approved color is red'

Thabo-hub committed:
GEN-845

'IM - Logbook - Approve Logbook -  button (Approve Logbook / Unapprove Logbook)   doesn’t change the colour, fixed now color is changing according to the status if approved color is green if not approved color is red'
